src/SignalR/clients/ts/signalr/src/LongPollingTransport.ts

```diff
@@ -111,13 +111,13 @@ export class LongPollingTransport implements ITransport {
             request.headers = {};
         }
         if (token) {
-            // tslint:disable-next-line:no-string-literal
+            // eslint-disable-next-line @typescript-eslint/dot-notation
             request.headers["Authorization"] = `Bearer ${token}`;
```

Comment:
Yeah I was not sure about these. We could change to use dot notation, like in this case it would work fine. In other places, we might need to add the type, like `const headers: MessageHeaders = {};`, so we can do `headers.Cookie = `${cookies}`;`. 

Another approach is have these header names as constants somewhere. Then it would not complain: 
`headers[COOKIE_HEADER] = `${cookies}`;` like so.

Reply:
Introduced a `HeaderNames` file containing only cookie and authorization. I guess we can add more as the need surfaces. Let me know what you think :)